lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -664,7 +749,7 @@
         const subset = [];
         _.forEach(requestedLocales, function (locale) {
             const noExtensionsLocale = parseLangtag(locale).base;
-            if (BestAvailableLocale(isAvailableLocale, locale) !== undefined) {
+            if (BestAvailableLocale(isAvailableLocale, noExtensionsLocale) !== undefined) {
```

Comment:
>noExtensionsLocale [](start = 55, length = 18)

was this overlooked in a previous change?

Reply:
Yep - the algorithm actually handles it just fine, this is just more efficient because we know availableLocales will never contain locales with any extensions.